doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -659,12 +659,14 @@ file is a particular case of a module called a *library file*.
 
 .. cmd:: Declare ML Module {+ @string }
 
-   Loads an OCaml plugin and its dependencies dynamically.  The :n:`@string`
-   argument must be a valid `findlib <http://projects.camlcity.org/projects/findlib.html>`_
-   plugin name, for example ``coq-core.plugins.ltac``. As of Coq 8.16,
-   the command also supports a legacy
-   syntax compatible with the plugin loading system used in Coq
-   8.0-8.15, see below.
+   Loads OCaml plugins and their dependencies dynamically.  The :n:`@string`
+   arguments must be valid `findlib <http://projects.camlcity.org/projects/findlib.html>`_
+   plugin names, for example ``coq-core.plugins.ltac``.
+   They must also contain a `.` to disambiguate against the removed non-findlib loading method.
+
+   Effects (such as adding new commands) from the explicitly requested
+   plugin are activated, but effects from implicitly loaded
+   dependencies are not activated.
```

Comment:
This is vague.

"effects" means adding or changing commands and tactics (and nothing else)?  If so, maybe say that and leave out "effect".  "effect" is vague, it basically just means a change.  Dictionary definition:

> Effect is usually a noun meaning "a change that results when something is done or happens,"

Also, what does "activated" mean here?  How do you later activate implicitly loaded dependencies--you need a separate `Declare ML Module`?  If so, say more explicitly.


How do you subsequently activate

Reply:
>"effect" is vague

plugins can do vague things so it's accurate